charon/src/translate/translate_ctx.rs

```diff
@@ -452,28 +452,43 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
             .iter()
             .any(|attr| attr == "charon::opaque" || attr == "aeneas::opaque");
         let rename = {
-            let pos = attributes.iter().position(|str| {
+            let str = attributes.iter().find(|str| {
                 str.starts_with("charon::rename(") || str.starts_with("aeneas::rename(")
             });
-            if pos.is_none() {
+            if str.is_none() {
```

Comment:
You can use the `if let ...` notation (you use it below actuallly) instead of `str.is_none()` then `str.unwrap()`.